src/components/PresentBar/PresentBar.test.tsx

```diff
@@ -252,4 +252,50 @@ describe('<PresentBar/>', () => {
       })
     ).toBeDisabled();
   });
+
+  it('should toggle the edit mode', async () => {
+    whiteboardManager
+      .getActiveWhiteboardInstance()
+      ?.getPresentationManager()
+      .startPresentation();
+
+    render(<PresentBar />, { wrapper: Wrapper });
+
+    const toolbar = screen.getByRole('toolbar', { name: 'Present' });
+
+    expect(
+      within(toolbar).getByRole('checkbox', {
+        name: 'Enable editing',
+        checked: false,
+      })
+    ).toBeInTheDocument();
```

Comment:
do we really have to check explicitly if the checkbox is in tzhe document, wouldnt it be sufficiant to directly wait for the user to click?!